components/02-content/footnotes/footnotes.hbs

```diff
@@ -0,0 +1,6 @@
+<p>In juli 2018 is er een quick scan natuur uitgevoerd voor het plangebied<sup id="#voetnoot14link"><a href="#voetnoot14" class="dso-footnote-link">[14]</a></sup>. Dit onderzoek is een actualisatie van een eerder door Blom Ecologie uitgevoerd oriënterend onderzoek dat vanwege de datum van uitvoering haar geldigheid was verloren. In januari 2019 is dit onderzoek aangevuld<sup id="#voetnoot15link"><a href="#voetnoot15" class="dso-footnote-link">[15]</a></sup> met vrije kavel, welke eveneens onderdeel zijn van dit bestemmingsplan.</p>
+
+<ol class="dso-footnotes">
+  <li class="dso-footnote" id="voetnoot14">14. <a href="#voetnoot14link" class="dso-footnote-return">^</a> Foreest Groen Consult (2018) Quickscan natuuronderzoek Ontwikkeling Herveld-Noord. 26 juli 2018.</li>
```

Comment:
`<a href="#voetnoot14link" class="dso-footnote-return">^</a>`  zegt natuurlijk niks, Daar moet wel een "terug naar leestekst" bij. Bv. als `aria-label`, of vinden we wellicht zelfs dat dit voor 'zienden' ook relevant is, dan wellicht gewoon een `title`?

Reply:
Wikipedia doet zelfs allebei. Omdat we gezegd hebben dat we zoveel mogelijk Wikipedia gaan nadoen, moeten wij dat ook maar doen:
```<span class="mw-cite-backlink"><a href="#cite_ref-1" aria-label="Omhoog" title="Omhoog">↑</a></span>```

Reply:
Wat denken jullie ervan om het `^`-character in een pseudo element content te zetten? Als er laten de wens is om hier een icoon voor te gebruiken kan dat vanuit de toolkit geregeld worden

Reply:
En ik vind `dso-footnote-return` geen goede modifier, wikipedia's `backlink` is duidelijker. Of een variant hierop